src/recorder.h

```diff
@@ -75,6 +79,12 @@ class Recorder {
   /// returns the unique id associated with this cyclus simulation.
   boost::uuids::uuid sim_id();
 
+  /// returns whether or not the unique simulation id will be injected.
+  bool inject_sim_id() { return inject_sim_id_; };
+
+  /// sets whether or not the unique simulation id will be injected.
+  void inject_sim_id(bool x) { inject_sim_id_ = x; };
```

Comment:
Just calling `set_dump_count()` will lose data, which is why I didn't do it. I'll `Flush()` and then set, though, as that is probably the safest option.